src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsSynchronisationService.kt

```diff
@@ -38,6 +45,9 @@ class NonAssociationsSynchronisationService(
   }
 }
 
+// Two non-association events occur for each non-association relationship - use the offenderIds to uniquely identify each pair
+private fun NonAssociationsOffenderEvent.isDuplicate(): Boolean = offenderIdDisplay > nsOffenderIdDisplay
```

Comment:
Not sure if this is a good name - possibly isNonAssociationIdentifier or isNonAssociationPrimary? all ideas welcome

Reply:
I think I prefer isNonAssociationPrimary because of the various drawbacks of using the term Duplicate in this scenario. 
I do wonder if the best thing is to deduplicate these events in prison-offender-events by using the alphabetical order trick. What do you think @andymarke?

Reply:
Hmm, not sure since "other" clients might want both events - in fact I think we raise to domain events which might be what clients want, so I think doing it here works slightly better.